pkg/virt-api/api.go

```diff
@@ -776,6 +778,10 @@ func (app *virtAPIApp) createValidatingWebhook() error {
 		}
 	} else {
 
+		// Ensure that we have the operator label attached, so that the operator can manage the resource later
+		// This is part of a soft transition from behing controlled by the apiserver, to being controlled by the operator
+		webhookRegistration.Labels[v1.ManagedByLabel] = v1.ManagedByLabelOperatorValue
```

Comment:
Are webhooks namespaced? won't we hit the namespaced owner which owns a non-namespaced resource issue?

Reply:
We only add a label, no owner references.

Reply:
Right! :smile: